d3a_interface/utils.py

```diff
@@ -427,3 +428,28 @@ def scenario_representation_traversal(sc_repr, parent=None):
             yield from scenario_representation_traversal(child, sc_repr)
 
     yield sc_repr, parent
+
+
+class HomeRepresentationUtils:
+    @staticmethod
+    def _is_home(representation):
+        home_devices = ["PV", "Storage", "Load", "MarketMaker"]
+        return all("type" in c and c["type"] in home_devices
+                   for c in representation["children"])
+
+    @classmethod
+    def is_home_area(cls, representation):
+        is_market_area = not key_in_dict_and_not_none(representation, "type") or representation["type"] == "Area"
+        has_home_devices = key_in_dict_and_not_none(representation, "children") and representation["children"] and \
```

Comment:
It is safe to omit the `and representation["children"]` check imo

Reply:
Safe but not optimal, if I omit it the _is_area will still be called with an empty list. However the impact is negligible, thus if you find it easier to read then let me know to remove it.

Reply:
It's okay then :)